modules/apiserversource-kn.adoc

```diff
@@ -13,38 +13,51 @@ This guide describes the steps required to create, manage, and delete an ApiServ
 
 .Procedure
 
-. Create an ApiServerSource event source by entering the following command:
+. Create an ApiServerSource that uses a broker as an event sink, by entering the following command:
 +
+
+[source,terminal]
 ----
-$ kn source apiserver create testevents --sink broker:default --resource "event:v1" --service-account events-sa --mode Resource
+$ kn source apiserver create <event-source-name> --sink broker:<broker-name> --resource "event:v1" --service-account <service-account-name> --mode Resource
```

Comment:
Use underscores and not dashes for any replaceable values in code blocks, for example `event_source_name>` instead of `<event-source-name>`.